docs/guides/marking-guide.md

```diff
@@ -0,0 +1,210 @@
+---
+id: marking-guide
+title: Marking Guide
+sidebar_label: Marking Guide
+---
+
+This Rubric should be used to guide your assessment of Students Homework at CodeYourFuture
+
+## Overview
+
+### Completeness (/10)
+
+- The code achieves all of the tasks in the coursework
```

Comment:
May be worth calling out something about it doing so in the intended way. Sometimes we see learners e.g. changing test cases, or over-fitting their solutions to the test cases in the homework - they should aim to identify and solve the general problem set by the task.